src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -147,11 +154,54 @@ public HttpRequestActionBuilder getDrugOrders(String patientUuid) {
 						"&careSetting=" + CARE_SETTING_UUID +
 						"&status=any&orderType=" + DRUG_ORDER +
 						"&v=" + customRepresentation);
-	}
+	}	
 	
-	public HttpRequestActionBuilder getAllergies(String patientUuid) {
+	public HttpRequestActionBuilder GetAllergies(String patientUuid) {
```

Comment:
```suggestion
	public HttpRequestActionBuilder getAllergies(String patientUuid) {
```

Method names should be named in lower camel case (starts with a lowercase).